src/components/publishing/__stories__/read_more.story.tsx

```diff
@@ -0,0 +1,114 @@
+import { storiesOf } from "@storybook/react"
+import { extend } from "lodash"
+import * as React from "react"
+import Article from "../article"
+import { StandardArticle } from "../fixtures/articles"
+import { RelatedCanvas, RelatedPanel } from "../fixtures/components"
+
+storiesOf("Publishing/Read More", module).add("Text Article", () => {
+  const article = extend({}, StandardArticle, {
+    sections: [
```

Comment:
Maybe it would make sense to move these articles into the fixtures file?  Seems like it would be useful to have more sample data options for testing in Positron and Force.

Reply:
Yeah, moving into fixtures seems like a good idea! Will do.